codegen/src/main/resources/core/src/api/ContractLifecycle.mustache

```diff
@@ -26,12 +25,19 @@ interface {{capitalizedContractName}}Lifecycle {
         summary = "Deploys the {{capitalizedContractName}} contract"
     )
     fun deploy{{deployParameters}}: TransactionReceipt
-    
+
     @Path(CONTRACT_ADDRESS_PATH)
     @Operation(
         tags = ["{{capitalizedContractName}} Methods"],
         summary = "Loads the {{capitalizedContractName}} contract"
     )
     fun load(@PathParam(CONTRACT_ADDRESS) contractAddress: String): {{capitalizedContractName}}Resource
+
+    @Path("")
```

Comment:
I believe we don't need the `Path` annotation here, maybe for Swagger purposes is needed.

Reply:
Swagger doesn't find this if we don't put that `@Path` in there and thus the endpoints exist, but they don't show up in the SwaggerUI